core/src/main/scala/eventstore/core/operations/UpdatePersistentSubscriptionInspection.scala

```diff
@@ -12,9 +13,9 @@ private[eventstore] final case class UpdatePersistentSubscriptionInspection(out:
 
   def decision(error: UpdatePersistentSubscriptionError) = {
     val result = error match {
-      case AccessDenied => new AccessDeniedException(s"Write access denied for stream $streamId")
-      case DoesNotExist => new InvalidOperationException(s"Subscription group ${out.groupName} on stream $streamId does not exist")
-      case Error(msg)   => new ServerErrorException(msg.orNull)
+      case AccessDenied => AccessDeniedException(s"Write access denied for stream $streamId")
+      case DoesNotExist => InvalidOperationException(s"Subscription group ${out.groupName} on stream $streamId does not exist")
+      case Error(msg)   => ServerErrorException(msg.orNull)
```

Comment:
are we sure about `.orNull` here ?

Reply:
No, could also do `.getOrElse(“”)`. Will look at this closer